diff --git a/node_modules/@radix-ui/react-popover/dist/index.mjs b/node_modules/@radix-ui/react-popover/dist/index.mjs
index 7a9697924d55c742ac971f11a43a0d165f121d59..2d24266133a8244e830fe3537e2832866952e46b 100644
--- a/node_modules/@radix-ui/react-popover/dist/index.mjs
+++ b/node_modules/@radix-ui/react-popover/dist/index.mjs
@@ -194,11 +194,11 @@ const PopoverContent = React.forwardRef((props, forwardedRef) => {
   const { __scopePopover, ...contentProps } = props;
   const context = usePopoverContext(CONTENT_NAME, __scopePopover);
-  return /* @__PURE__ */ React.createElement(Presence, {
-    present: context.open
+  return context.open ? /* @__PURE__ */ React.createElement(Presence, {
+    present: true
   }, /* @__PURE__ */ React.createElement(PopoverContentImpl, {
     ...contentProps,
     ref: forwardedRef
-  }));
+  })) : null;
 });
 PopoverContent.displayName = CONTENT_NAME;
 

